GLAA.ViewModels/Admin/AdminLicenceViewModel.cs

```diff
@@ -19,16 +29,36 @@ public class AdminLicenceViewModel : PleaseSelectViewModel
 
         public ICollection<LicenceStatusViewModel> LicenceStatusHistory { get; set; }
 
-        public LicenceStatusViewModel LatestStatus => LicenceStatusHistory.OrderByDescending(l => l.DateCreated).First();
-
-        public IEnumerable<SelectListItem> NextStatusDropDown => PleaseSelectItem.Concat(LatestStatus.NextStatuses.Select(s => s.DropDownItem));
+        public LicenceStatusViewModel LatestStatus => LicenceStatusHistory?.OrderByDescending(l => l.DateCreated).First();
 
-        public List<CheckboxListItem> StandardCheckboxes { get; set; }
+        public IEnumerable<SelectListItem> NextStatusDropDown => LatestStatus != null ? PleaseSelectItem.Concat(LatestStatus.NextStatuses?.Select(s => s.DropDownItem)) : null;        
```

Comment:
Now I think about it, we can get rid of `PleaseSelectItem` as there's an overload for `HtmlHelper.DropDownListFor` which allows you to specify a default item.

Reply:
Nice, that'd certainly save some complexity